docs/nodes/nodes-library/nodes/GitHub/README.md

```diff
@@ -0,0 +1,63 @@
+---
+permalink: /nodes/n8n-nodes-base.github
+---
+
+# GitHub
+
+[GitHub](https://github.com/) provides hosting for software development and version control using Git. It offers the distributed version control and source code management (SCM) functionality of Git, access control and several collaboration features such as bug tracking, feature requests, task management, and wikis for every project.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Github/README.md).
+:::
+
+## Basic Operations
+
+- File
+    - Create a new file in repository
+    - Delete a file in repository
+    - Edit a file in repository
+    - Get the data of a single issue
+
+- Issue
+	- Create a new issue
+	- Create a new comment on an issue
+	- Edit an issue
+	- Get the data of a single issue
+	- Lock an issue
+
+- Repository
+	- Get the data of a single repository
+	- Return the contents of the repository's license file, if one detected
```

Comment:
Could you change it to match the node?
![image](https://user-images.githubusercontent.com/48641776/86738717-ef212700-c052-11ea-895a-03ecc1263756.png)